src/styles/main_page/_GongZhongHao.scss

```diff
@@ -0,0 +1,37 @@
+//@import "../../main_page/GongZhongHao.js";
+
+.MainGrid { 
+    display: grid;
+    //grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+    //grid-template-columns: 400px 400px 400px;
+    grid-template-columns: repeat(auto-fill, 370px);
+    grid-template-rows: repeat(auto-fill, 290px);
+    grid-gap: 10px;
+    align-items: stretch;
+    padding-left: 350px;
+    padding-top: 150px;
+    }
+  .MainGrid > article {
+    border: 1px solid #ccc;
+    box-shadow: 2px 2px 6px 0px  rgba(0,0,0,0.3);
+  }
+  .MainGrid > article img {
+    //max-width: 80%;
+    width: 370px; /* width of container */
+    height: 200px; /* height of container */
+    object-fit: cover;
+    border: 2px solid rgba(224, 220, 220, 0.795);
+  }
+  .MainText {
+    padding: 0 20px 20px;
+    font-size: 13px;
+    font-family: "Arial","Microsoft YaHei","黑体","宋体",sans-serif;
+    letter-spacing: 2px;
+  }
+  .MainText > button {
+    background: gray;
+    border: 0;
+    color: white;
+    padding: 10px;
+    width: 100%;
```

Comment:
We want the website to be responsive, so you need to implement responsive styles.